app/helpers/navigationHelpers.php

```diff
@@ -633,12 +633,10 @@ function _caNavTypeToImgName($pn_type) {
 				break;
 			case __CA_NAV_BUTTON_COLLAPSE__:
 				$vs_img_name = 'glyphicons_191_circle_minus';
-				$vs_classname = 'collapse';
```

Comment:
I added these class names in the previous PR, but they are unnecessary now since they are hardcoded in the view.  Removing to avoid the unnecessary delta.